tests/UnitTests/TimerCollectionTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Threading;
+using Spiffy.Monitoring;
+using Kekiri.Xunit;
+using Xunit;
+
+namespace UnitTests
+{
+    public class TimerCollectionTests : Scenarios
+    {
+        [Scenario]
+        public void TimeOnceScenario()
+        {
+            Given(A_code_block_timed_once);
+            When(Event_is_logged);
+            Then(The_time_elapsed_field_should_be_near, 10)
+                .And(There_should_be_no_count_field);
+        }
+
+        [Scenario]
+        public void AccumulateScenario()
+        {
+            Given(A_code_block_timed_multiple_times);
+            When(Event_is_logged);
+            Then(The_time_elapsed_field_should_be_near, 20)
+                .And(There_should_be_a_count_field);
+        }
+
+        void A_code_block_timed_once()
+        {
+            using (EventContext.Timers.Replace(TimerKey))
```

Comment:
I'm not in love with the "Replace" semantic.  Possible alternatives:
* `Time` / `TimeOnce`
* `Start` / `StartNew`
* something else

Reply:
`StartOrReplace`?

Reply:
Went with `TimeOnce`